src/k8s/api/v1/cluster_node.go

```diff
@@ -1,11 +1,11 @@
 package v1
 
-// JoinNodeRequest is used to request to add a node to the cluster.
-type JoinNodeRequest struct {
+// JoinClusterRequest is used to request to add a node to the cluster.
```

Comment:
Thanks for the catch!
Btw, we probably need to rename things here soon again:
Looking at the CLI spec, it seems like we go with `k8s add` and `k8s join` now.